facade/src/main/java/org/jboss/pnc/facade/providers/BuildProviderImpl.java

```diff
@@ -67,20 +69,27 @@
     private BuildConfigurationAuditedRepository buildConfigurationAuditedRepository;
     private Gerrit gerrit;
     private BuildConfigurationRevisionMapper buildConfigurationRevisionMapper;
+    private BuildMapper buildMapper;
```

Comment:
The mapper is already included in the superclass. I think it's named `mapper`.

Reply:
I cannot use the one in the superclass since it's an EntityMapper type, with `toDTO` and other entity + DTO conversions.